main/templates/admin/test/test_versions.html

```diff
@@ -0,0 +1,22 @@
+<p>
+  Modules and their versions, '*' indicates those in the active working set.
+  <div class="alert alert-warning {{'hide' if versions}}">
+    No data on versions
+  </div>
```

Comment:
[`div` cannot be inside `p`](https://stackoverflow.com/a/8696078/8418)

put it in a separate block

Reply:
Okay... thought I had tested this.